### PR TITLE
minor: stop warning that AQEShuffleRead cannot run natively

### PR DESCRIPTION
**Pull request by andygrove**
## Which issue does this PR close?

N/A

## Rationale for this change

`AQEShuffleRead` will never have a native version so we should stop warning that we cannot run it natively

### CODE REVIEW

File: spark/src/main/scala/org/apache/comet/CometSparkSessionExtensions.scala [hunk 1]
```diff
@@ -826,7 +826,7 @@ class CometSparkSessionExtensions
           op match {
             // Broadcast exchange exec is transformed by the parent node. We include
             // this case specially here so we do not add a misleading 'info' message
-            case _: BroadcastExchangeExec => op
+            case _: BroadcastExchangeExec | _: AQEShuffleReadExec => op
```
Comment (parthchandra) [inline spark/src/main/scala/org/apache/comet/CometSparkSessionExtensions.scala:829]: Update the comment as well?

**Review by parthchandra — APPROVED** [review]
LGTM (with minor comment)

File: spark/src/main/scala/org/apache/comet/CometSparkSessionExtensions.scala [hunk 2]
```diff
@@ -226,7 +226,12 @@ class CometSparkSessionExtensions
 
     private def isCometPlan(op: SparkPlan): Boolean = op.isInstanceOf[CometPlan]
 
-    private def isCometNative(op: SparkPlan): Boolean = op.isInstanceOf[CometNativeExec]
+    private def isCometNative(op: SparkPlan): Boolean = op match {
+      case _: CometNativeExec => true
+      case AQEShuffleReadExec(q: ShuffleQueryStageExec, _) =>
+        isCometNative(q.plan)
```
Comment (viirya) [inline spark/src/main/scala/org/apache/comet/CometSparkSessionExtensions.scala:232]: Hmm, `ShuffleQueryStageExec`'s child must be a `ShuffleExchangeLike` or `ReusedExchangeExec`. In Comet query, it should be Comet shuffle which is `CometPlan`.

Did you see there is a `CometNativeExec` under `ShuffleQueryStageExec`?

**Review by viirya — APPROVED** [review]

**Review by kazuyukitanimura — APPROVED** [review]

**Comment by viirya** [comment]
Thanks @andygrove @parthchandra @kazuyukitanimura

**Pull request merged**